types/react-native/tsconfig.json

```diff
@@ -5,6 +5,7 @@
             "es6",
             "dom"
         ],
+        "declaration": true,
```

Comment:
I just add it for running test case, since it seems to this error only happens when  emitting declaration files. see https://github.com/Microsoft/TypeScript/issues/9944